Buildings/Resources/src/ThermalZones/Detailed/Validation/BESTEST/simulateAndPlot.py

```diff
@@ -412,6 +454,219 @@ def _find_data_bin(dataSet):
         results[i] = '{}'.format(int(counter))
     return results
 
+def export_data(data_set):
+    KWh = 3600*1e3
```

Comment:
For consistency I would use lower case `kWh` as kilo is abbreviated lower case.

Reply:
changed.